rust/src/transformations/make_stable_expr/expr_count/pseudocode/counting_query_stability_map.py

```diff
@@ -0,0 +1,23 @@
+# type: ignore
+def counting_query_stability_map(
+    public_info: Literal["Keys"] | Literal["Lengths"] | None,
+) -> StabilityMap[PartitionDistance[M], LpDistance[P, f64]]:
+
+    if public_info == "Lengths":  # `\label{public-info}`
+        return StabilityMap.new(lambda _: 0.)
+        
+    def norm_map(d_in: f64) -> f64:  # `\label{norm-map}`
+        if P == 1:
+            return d_in
+        if P == 2:
+            return d_in.inf_sqrt()
```

Comment:
If I'm reading this as python, it wouldn't work, because type parameters aren't available to runtime code. I might be misreading this.